README.md

```diff
@@ -1,2 +1,46 @@
 # container-ipam-tool
 Configuration tool for any container environments
+
+## Prerequisite
+
```

Comment:
A NIOS DDI Appliance with cloud automation License. 

you can download a virtual version of the product from the Infoblox Download Center (https://www.infoblox.com/infoblox-download-center). Alternatively, if you are an existing Infoblox customer, you can download it from the support site.